package.json

```diff
@@ -27,7 +27,7 @@
   },
   "scripts": {
     "start": "npm run test:watch",
-    "test": "cross-env NODE_ENV=test NODE_PATH=./src babel-node node_modules/.bin/tape test/*.spec.js test/**/*.spec.js | tap-spec",
+    "test": "cross-env NODE_ENV=test NODE_PATH=./src babel-node test/index.js | tap-spec",
```

Comment:
@emmenko Why can't we keep the pattern here, so that all our test files are detected and executed automatically? I think it is very convenient the way it is.
What is the advantage of maintaining a list of all tests in the index file?